content/docs/started/getting-started-k8s.md

```diff
@@ -144,11 +144,342 @@ kubectl get cm dex -n kubeflow -o jsonpath='{.data.config\.yaml}' > dex-config.y
+
+    1. Get the current Dex config from the corresponding Config Map.
+        
+        {{< highlight bash >}}
+        kubectl get configmap -n kubeflow -o jsonpath='{.data.config\.yaml}' > dex-config.yaml
+        {{< /highlight >}}
+
+    1. Add the LDAP-specific options. Here is an example to help you out. It is configured to work with the example LDAP Server you set up previously.
+        
+        <details>
+        
+        <summary>Dex LDAP Config Section</summary>
+        {{< highlight yaml >}}
+        connectors:
+        - type: ldap
+          # Required field for connector id.
+          id: ldap
+          # Required field for connector name.
+          name: LDAP
+          config:
+            # Host and optional port of the LDAP server in the form "host:port".
+            # If the port is not supplied, it will be guessed based on "insecureNoSSL",
+            # and "startTLS" flags. 389 for insecure or StartTLS connections, 636
+            # otherwise.
+            host: ldap-service.kubeflow.svc.cluster.local:389
+          
+            # Following field is required if the LDAP host is not using TLS (port 389).
+            # Because this option inherently leaks passwords to anyone on the same network
+            # as dex, THIS OPTION MAY BE REMOVED WITHOUT WARNING IN A FUTURE RELEASE.
+            #
+            insecureNoSSL: true
+          
+            # If a custom certificate isn't provide, this option can be used to turn off
+            # TLS certificate checks. As noted, it is insecure and shouldn't be used outside
+            # of explorative phases.
+            #
+            insecureSkipVerify: true
+          
+            # When connecting to the server, connect using the ldap:// protocol then issue
+            # a StartTLS command. If unspecified, connections will use the ldaps:// protocol
+            #
+            startTLS: false
+          
+            # Path to a trusted root certificate file. Default: use the host's root CA.
+            # rootCA: /etc/dex/ldap.ca
+            # clientCert: /etc/dex/ldap.cert
+            # clientKey: /etc/dex/ldap.key
+          
+            # A raw certificate file can also be provided inline.
+            # rootCAData: ( base64 encoded PEM file )
+          
+            # The DN and password for an application service account. The connector uses
+            # these credentials to search for users and groups. Not required if the LDAP
+            # server provides access for anonymous auth.
+            # Please note that if the bind password contains a `$`, it has to be saved in an
+            # environment variable which should be given as the value to `bindPW`.
+            bindDN: cn=admin,dc=arrikto,dc=com
+            bindPW: admin
+          
+            # The attribute to display in the provided password prompt. If unset, will
+            # display "Username"
+            usernamePrompt: Arrikto username
```

Comment:
Same comment about the use of "Arrikto" in this section - is it just an example, or is it required because the user is using Arrikto software?